src/app/window-handler.ts

```diff
@@ -1035,6 +1042,114 @@ export class WindowHandler {
         focusedWindow.webContents.setZoomLevel(currentZoomLevel + 0.5);
     }
 
+    /**
+     * HACK SWITCH to Client 1.5
+     */
+    private async onClient1_5(): Promise <void> {
+        logger.info('window handler: go to Client 1.5');
+        logger.info('this.globalConfig.url: ' + config.getGlobalConfigFields([ 'url']));
+
+        if (config.getGlobalConfigFields([ 'url']).url && config.getGlobalConfigFields([ 'url']).url.startsWith('https://corporate.symphony.com')) {
+            logger.info('window handler: switchClient: ' + switchClient);
+            if (switchClient === 'client1_5' ) {
```

Comment:
I think it's better to rename as currentClient

Reply:
done